### PR TITLE
Wrk 41 active contracts only

### PR DESCRIPTION
**Pull request by Fputker**
Nieuw scherm voor Actieve contracten

### CODE REVIEW

**Review by timsmelik — CHANGES_REQUESTED** [review]

File: src/main/react/clients/ContractClient.ts [hunk 1]
```diff
@@ -29,44 +29,44 @@ export type ContractRaw = {
   type: "INTERNAL" | "EXTERNAL" | "MANAGEMENT" | "SERVICE";
 };
 
-export type ContactRequest = {};
+export type ContractRequest = {};
```
[the remark this answers is not in the document]
Comment (Fputker) [reply src/main/react/clients/ContractClient.ts:32]: The only thing I fixed is the spelling. The lack of types inside has been there for a while...

I guess the only common thing in the contract request is a `from` and a nullable `to`. We could add those here, but I am not sure what the repercussions are.

File: src/main/react/features/person/table/PersonTableHead.tsx [hunk 2]
```diff
@@ -10,26 +10,26 @@ const headCells = [
   },
   {
     id: "email",
-    algin: false,
+    align: false,
```
[the remark this answers is not in the document]
Comment (Fputker) [reply src/main/react/features/person/table/PersonTableHead.tsx:13]: true, done.

File: src/main/react/features/report/Contract/ContractReportTable.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+  const handleRowsPerPageChange = (event) => {
+    const rowsPerPage = event.target.value;
+    setSize(rowsPerPage);
+    setPage(0);
+  };
+
+  return (
+    <>
+      <CardHeader
```
[the remark this answers is not in the document]
Comment (Fputker) [reply src/main/react/features/report/Contract/ContractReportTable.tsx:61]: Removing the tag from around tge <CardHeader> tag gives me the following error: 
TS2657: JSX expressions must have one parent element.

File: src/main/react/features/report/Contract/ContractReport.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {Card, CardContent, Grid} from "@material-ui/core";
+import ContractReportTable from "./ContractReportTable";
+
+export default function ContractReport() {
+  return <Grid item xs={12}>
```
[the remark this answers is not in the document]
Comment (Fputker) [reply src/main/react/features/report/Contract/ContractReport.tsx:6]: done

File: src/main/react/features/report/Contract/ContractReportTableHead.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import {TableCell, TableHead, TableRow} from "@material-ui/core";
+import React from "react";
+
+const headCells = [
+  {
+    id: "person",
+    align: false,
```
[the remark this answers is not in the document]
Comment (Fputker) [reply src/main/react/features/report/Contract/ContractReportTableHead.tsx:7]: done

File: src/main/react/features/report/Contract/ContractReportTable.tsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+              );
+            })}
+          </TableBody>
+        </Table>
+        <TablePagination
+          rowsPerPageOptions={[10, 20, 50, {value: -1, label: 'All'}]}
+          component="div"
+          count={rowCount}
+          // remove labelDisplayRows by replacing it with an empty return
+          // labelDisplayedRows={() => null}
```
[the remark this answers is not in the document]
Comment (Fputker) [reply src/main/react/features/report/Contract/ContractReportTable.tsx:97]: done

**Review by wilmveel — APPROVED** [review]

File: src/main/react/features/report/Contract/ContractReportTable.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Fputker) [reply src/main/react/features/report/Contract/ContractReportTable.tsx]: Willem explained what you mean. I've updated this

**Pull request merged**